### DISCUSSION TITLE
How to delete the docker image and start fresh?

### DISCUSSION THREAD

**Discussion by jukofyork** [open-webui/open-webui#4740] [General]
All I want to do is delete it and start again due to having 100s of models I got when I linked my openrouter key...

```sh
docker ps
docker stop <...>
docker system prune -a -f --volumes
```

```sh
Total reclaimed space: 3.899GB
```

So I then run:

```sh
docker run -d --network=host -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```

and it redownloads several GBs of data, but the data all seems to be retained (somewhere???) and I can still log in and everything works as before (including the 100s of models I got when I linked my openrouter key).

So where the heck is the data being stored?

**Comment by jukofyork** [comment]
I managed to fix it by purging and uninstalling everything to do with docker.

Luckily I don't use it for anything else (thank god!!!) so it doesn't really matter, but I don't get why everything to do with docker just has to turn into a complete clownshow like this... :/

**Reply by WolCarlos** [reply]
Hello,
I had the same problem and solved it by using Everithing to search all openwebUI files and delete them.
I did not have to uninstall Docker.
Before doing this I deleted through Docker the conteiner, the image and the 
Volume.
![image](https://github.com/user-attachments/assets/50e4e33d-f27a-4e02-80cd-9e0fa01ead6f)
![image](https://github.com/user-attachments/assets/0904e2ed-6ea1-4e62-9e75-ec0e6442f007)
![image](https://github.com/user-attachments/assets/80e2fc1f-3c33-4350-93cf-aacfa71b9b1f)
